libwallet/assets/wallet/wallet_utils.go

```diff
@@ -133,7 +132,7 @@ func (wallet *Wallet) VerifySeedForWallet(seedMnemonic, privpass string) (bool,
 	wallet.mu.RLock()
 	defer wallet.mu.RUnlock()
 
-	decryptedSeed, err := decryptWalletSeed([]byte(privpass), wallet.EncryptedSeed)
+	decryptedSeed, err := decryptWalletMnemonic([]byte(privpass), wallet.EncryptedSeed)
```

Comment:
Same as my comment above, but can ignore for `wallet.EncryptedSeed`, cuz it'll break existing wallets.